guides/examples.md

```diff
@@ -0,0 +1,3 @@
+## two classes
+
+## `data-height`
```

Comment:
I may remove this OR flesh it out (if time permits). It was accidentally committed as I created it during a shadowing session with @advl . It would be nice to better document the examples API with its own guide file.